packages/react-liveness/src/components/FaceLivenessDetector/service/utils/streamProvider.ts

```diff
@@ -56,14 +58,16 @@ export class LivenessStreamProvider extends AmazonAIInterpretPredictionsProvider
     sessionId: string,
     region: string,
     stream: MediaStream,
-    videoEl: HTMLVideoElement
+    videoEl: HTMLVideoElement,
+    credentialProvider?: CredentialProvider
```

Comment:
This is a lot of positional arguments, can we just convert to an object and remove the "max-params" eslint-disable line?